integration-tests/api/__tests__/admin/shipping-options.js

```diff
@@ -123,7 +123,9 @@ describe("/admin/shipping-options", () => {
         })
 
       expect(res.status).toEqual(400)
-      expect(res.data.message).toEqual("ID does not exist")
+      expect(res.data.message).toEqual(
+        "Shipping option requirement with id not_allowed does not exist"
```

Comment:
self-review: The error message has been improved. With the new version of typeorm, the add requirement has been changed a bit